paimon-core/src/main/java/org/apache/paimon/utils/StatsUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.apache.paimon.utils;
+
+import org.apache.paimon.CoreOptions;
+import org.apache.paimon.options.Options;
+import org.apache.paimon.statistics.Stats;
+
+import java.util.List;
+
+import static org.apache.paimon.CoreOptions.FIELD_STATS_MODE_PREFIX;
+import static org.apache.paimon.CoreOptions.FIELD_STATS_MODE_SUFFIX;
+import static org.apache.paimon.options.ConfigOptions.key;
+
+/** The stats utils. */
+public class StatsUtils {
+
+    public static Stats[] getFieldsStatsMode(CoreOptions options, List<String> fields) {
```

Comment:
Can this method just in `Stats` class?

Reply:
It's mainly for the access of `CoreOptions` and `Options`